internal/converter/converter_test.go

```diff
@@ -122,152 +122,157 @@ func testConvertSampleProto(t *testing.T, sampleProto sampleProto) {
 
 func configureSampleProtos() map[string]sampleProto {
 	return map[string]sampleProto{
```

Comment:
I commented this all out for now - will uncomment and modify other tests once this PRs overall concept is approved